From 549dc616678478ee92b7a34a42e24a610ca81631 Mon Sep 17 00:00:00 2001
From: PonyWarrior <lob.revolu@gmail.com>
Date: Thu, 13 Feb 2020 21:43:43 +0800
Subject: [PATCH] DeathSkip

---
 DeathLoop.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DeathLoop.lua b/DeathLoop.lua
index 0ff1435..8cf74f7 100644
--- a/DeathLoop.lua
+++ b/DeathLoop.lua
@@ -59,7 +59,7 @@ function HandleDeath( currentRun, killer, killingUnitWeapon )
 
 	ZeroSuperMeter()
 	FinishTargetMarker( killer )
-	DeathPresentation( currentRun, killer, killingUnitWeapon )
+	--DeathPresentation( currentRun, killer, killingUnitWeapon )
 
 	currentRun.CurrentRoom.EndingHealth = currentRun.Hero.Health
 	currentRun.EndingMoney = currentRun.Money
-- 
2.25.0.windows.1

